### ISSUE TITLE
Context menus in Sublime 3 unintentionally mask text

### ISSUE THREAD

**Issue by thecjharries** [Monokai/monokai-pro-sublime-text#33]
![issue png](https://user-images.githubusercontent.com/6250642/30782150-08ccf400-a0f2-11e7-8836-a45b75384b00.png)
The context highlight is apparently the same color as the non-emphasized text, which has quickly become a serious problem. I use those menus more than I realized.

**Comment by thecjharries** [comment]
I did a few things to check this out.

* I was running 1.1.0, so I removed and reinstalled. Same issue.
* I share config between a few machines, so I removed my sync folder and used pure local. Same issue.
* I disabled SublimeLinter. Same issue.
* I read the docs and realized `Color Scheme !== Theme`. Issue solved.

So, the issue is that I was using a Monokai Pro Color Scheme, not a Monokai Pro Theme
![tooltip_206](https://user-images.githubusercontent.com/6250642/31182140-ad755454-a8e8-11e7-8589-b5efb6d54ff1.png)
Once I enabled the theme, not the color scheme, the context menu has yellow highlights instead of grey.
![tooltip_208](https://user-images.githubusercontent.com/6250642/31182211-de274440-a8e8-11e7-95da-955f273190da.png)
I honestly did not realize there was a difference. It seems that Monokai Pro doesn't have all the right bits for a color scheme, but, as the docs say, it's a theme.
